core/server/api/v2/utils/serializers/output/config.js

```diff
@@ -1,9 +1,9 @@
-const debug = require('ghost-ignition').debug('api:v2:utils:serializers:output:configuration');
+const debug = require('ghost-ignition').debug('api:v2:utils:serializers:output:config');
 
 module.exports = {
-    all(configuration, apiConfig, frame) {
+    all(data, apiConfig, frame) {
         frame.response = {
-            configuration: configuration ? [configuration] : []
+            config: data
```

Comment:
Decided to expose the "resource-type" in the body

```
config: {version: ....}
```

```
site: {version: ....}
```

Reasons:
- the API always returns the resource type
- the SDK would need to add an exception if we do not expose the resource type

Feel free to update the endpoint to return an array with one element, I decided against, because it's so uncomfortable returning this format.

@gargol I think we should expose the resource type for the images endpoints too? Otherwise, it's inconsistent.